src/query/frame_change.hpp

```diff
@@ -88,14 +103,14 @@ class FrameChangeCollector {
   // Reset value for tracking key
   bool ResetTrackingValue(const std::string &key) {
     if (tracked_values_.contains(key)) {
-      tracked_values_[key].cache_.clear();
+      tracked_values_.at(key).cache_.clear();
     }
 
     return true;
   }
 
   // Get value cached for tracking key
-  CachedValue &GetCachedValue(const std::string &key) { return tracked_values_[key]; }
+  CachedValue &GetCachedValue(const std::string &key) { return tracked_values_.at(key); }
```

Comment:
This can throw since you can (in theory) receive any key. Is that ok?

Reply:
I think it is okay since there is interface to check whether value exists, I will just add comment that function may throw.